features/FEATURE_BLE/ble/Gap.h

```diff
@@ -2281,7 +2286,7 @@ class Gap {
      * @param[in] ownAddr Address this device uses for this connection.
      * @param[in] connectionParams Parameters of the connection.
      */
-    void processConnectionEvent(
+    virtual void processConnectionEvent(
```

Comment:
virtuality qualification  should not be changed for the sake of an implementation detail.

Reply:
removed
(it is still irrecoverable and it is being overridden but I take your point as this is a hint to the porter but then again we have explicit comments for porters)